backend/src/utils/route-security.ts

```diff
@@ -70,6 +44,28 @@ const testAdmin = async (
   return false; // Not admin, no bypassing
 };
 
+const requestSecurityGuardNotebook = async (
```

Comment:
No, this is for other purposes, the username field is for the ability to spawn other users' notebooks. If not present, it will get the username by the request. If added, admins can spanw notebooks for existing users and regular users will have a returning error.